docs/proposals/node-agent/workload.md

```diff
@@ -0,0 +1,23 @@
+# Workloads
+
+The node agent amanges workload management. A workload is the tiniest unit in orka. It can be a container, a virtual machine, etc.
+
+For simplicity's sake we will suppose that we are dealing with container
+workloads.
```

Comment:
```suggestion
At the moment, `orka` only supports container workloads.
```

Reply:
Fixed in https://github.com/dev-sys-do/orka/pull/1/commits/162a3a8f1cd6aba48c05413934d6f8ad29df369f